bluepyopt/ephys/protocols.py

```diff
@@ -270,15 +283,23 @@ def _reduce_method(meth):
                                        sim=sim)
         return responses
 
-    def instantiate(self, sim=None, icell=None):
+    def instantiate(self, sim=None, cell_model=None):
         """Instantiate"""
 
         for stimulus in self.stimuli:
-            stimulus.instantiate(sim=sim, icell=icell)
+            if isinstance(stimulus, LFPStimulus):
+                stimulus.instantiate(lfpy_cell=cell_model.lfpy_cell)
```

Comment:
Should be sim, lfpy_cell

Reply:
@AurelienJaquier can you take care of it?

Reply:
> @AurelienJaquier can you take care of it?

actually I'm on it ;)